kubernetes/spilo.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: &cluster_name zalandodemo01 
+  labels:
+    application: spilo
+    spilo-cluster: *cluster_name
+spec:
+  replicas: 3
+  serviceName: *cluster_name
+  template:
+    metadata:
+      labels:
+        application: spilo
+        spilo-cluster: *cluster_name
+      annotations:
+        iam.amazonaws.com/role: app-spilo
+        scheduler.alpha.kubernetes.io/affinity: >
+            {
+                "podAntiAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": [
+                    {
+                        "labelSelector": {
+                          "matchExpressions": [
+                            {
+                              "key": "spilo-cluster",
+                              "operator": "In",
+                              "values": ["zalandodemo01"]
+                            }
+                          ]
+                         },
+                         "topologyKey": "kubernetes.io/hostname" 
+                    }
+                  ]
+                 }
+            }
+    spec:
+      serviceAccountName: operator
```

Comment:
Hm, probably not. I don't want to remove it, but I will add a comemnt, explaining, what does it do and why do we need it.